tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -434,6 +434,10 @@ def compute_nodes_count(self):  # noqa: D102
         # split()[-1] to extract last line and trim whitespaces
         return int(result.stdout.split()[-1])
 
+    def compute_nodes_count_wrapper(self):  # noqa: D102
+        result = self._remote_command_executor.run_remote_command("echo $(/opt/slurm/bin/pbsnodes | grep ip | wc -l)")
```

Comment:
I don't like that we are using a `/opt/slurm/bin/` command path in the `TorqueCommands` class. When calling the function from outside you should not have to decide which function to call (`compute_nodes_count` or `compute_nodes_count_wrapper`) base on the running scheduler.
Is it possible to remove the full command path from the `compute_nodes_count` function and just leave the command `pbsnodes` so that the function can be called indifferently from Slurm and from Torque case?